package.json

```diff
@@ -1,21 +1,25 @@
 {
   "name": "ddts",
-  "version": "1.4.0",
-  "main": "tslint.json",
+  "version": "2.0.0",
+  "main": ".eslintrc.json",
   "scripts": {
-    "postinstall": "npm run compilerules; ./install-hooks",
-    "compilerules": "$(git rev-parse --show-toplevel)/node_modules/.bin/tsc ./rules/*.ts || true"
+    "postinstall": "./install-hooks"
   },
   "description": "A typescript style guide with teeth",
   "author": "Connor Harwood <charwood@dronedeploy.com>",
   "license": "MIT",
   "devDependencies": {
-    "tslint": "^5.12.1"
+    "eslint": "^7.32.0"
   },
   "peerDependencies": {
-    "lint-staged": "^7.2.0",
-    "prettier": "^1.19.1",
-    "tslint": "^5.12.1",
+    "@typescript-eslint/eslint-plugin": "4.33.0",
+    "eslint": "^7.32.0",
+    "eslint-config-prettier": "^8.3.0",
+    "eslint-plugin-ban": "^1.6.0",
+    "eslint-plugin-import": "2.25.4",
+    "eslint-plugin-prefer-arrow": "1.2.2",
```

Comment:
I moved those packages to `peerDependencies` but maybe they should be put in `dependencies` instead? @CodyCahoon what do you think?